deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
+UPDATE button SET flavor_text="Shepherd has made ignorance of obvious things his lifelong ambition. He likes spoons, forgetting how to do stuff, and beating people up." WHERE name="Shepherd";
+UPDATE button SET flavor_text="Strik becomes more accustomed to his new lifestyle with each passing day. He likes quiet walks in the moonlight, bashing things with his head, and beating people up." WHERE name="Strik";
+UPDATE button SET flavor_text="Wastenott is dead, dead, dead, dead. He likes getting out once in a while, talking to old friends and beating people up." WHERE name="Wastenott";
+UPDATE button SET flavor_text="Al-Khwarizmi wrote an amazing book about solving everyday problems in life and business with mathematics. Our word \"algebra\" came from the title of his book, and the term \"algorithm\" is from a Latin translation of his name." WHERE name="Al-Khwarizmi";
+UPDATE button SET flavor_text="Best known for his Theory of Numbers, Carl Friedrich Gauss startled his grade school teacher by deducing that the sum of the integers from 1 to 100 was really the sum of 50 pairs of integers which all add to 101." WHERE name="Carl Friedrich Gauss";
+UPDATE button SET flavor_text="Before becoming the Akami Professor of Internet Mathematics at UC San Diego, Dr. Fan Chung Graham directed math research labs at Bellcore. She and her husband Ron developed semi-random routing theories that keep the Internet running." WHERE name="Fan Chung Graham";
+UPDATE button SET flavor_text="Leonard Eugene Dickson loved pure mathematics. He wrote 267 research papers and 18 books, including the three volume \"History of the Theory of Numbers.\" His favorite quote: \"Thank God that number theory is unsullied by any applications.\"" WHERE name="Leonard Eugene Dickson";
+UPDATE button SET flavor_text="One of the best known and most respected mathematicians of the 20th century, Paul Erdos (air-dish) wrote or co-authored 1,475 papers. As he put it, \"A mathematician is a machine for turning coffee into theorems.\"" WHERE name="Paul Erdos";
+UPDATE button SET flavor_text="Pythagoras and his team in Italy called the mathematikoi developed cool triangle theorems, described the five regular solids, and thought that numbers had personal characteristics like male, female, beautiful, and ugly." WHERE name="Pythagoras";
+UPDATE button SET flavor_text="Known informally as \"The Mathemagician,\" Dr. Ron Graham was Chief Scientist at AT&T Labs. He's treasurer of the National Academy of Sciences, and also the ex-president of the International Jugglers Association." WHERE name="Ron Graham";
+UPDATE button SET flavor_text="Theano worked alongside her husband, Pythagoras, writing about mathematics, physics, medicine, and child psychology. She's best known for her work on the \"Golden Mean\" and the \"Golden Rectangle.\"" WHERE name="Theano";
```

Comment:
I think it's a shame that the Button Brains buttons don't mention beating people up.  If you're not up for writing text at this time, let's open a separate issue to fix it.

(I mean, in actual point of fact, the Button Brains buttons do not excel at beating people up.  But i don't think flavor text should be a function of win percentage.)